ui/packages/consul-ui/app/templates/dc/kv/edit.hbs

```diff
@@ -121,17 +131,21 @@ as |parts|}}
         }}
         @onchange={{action (mut session) value="data"}}
       />
-  {{#if session}}
-      {{!FIXME}}
+      <h2>
+        <Action
+          rel="help"
+          @href={{concat (env 'CONSUL_DOCS_URL') '/internals/sessions.html#session-design'}}
+          @external={{true}}
+        >
+          Lock Session
+        </Action>
+      </h2>
+  {{#if session.ID}}
       <Consul::LockSession::Form
         @item={{session}}
-        @dc={{route.params.dc}}
-        @nspace={{route.params.nspace}}
-        @partition={{route.params.partition}}
-        @onsubmit={{action (noop) undefined}}
```

Comment:
We reference all of the 'buckets' from the session itself now so we don't need to pass these through individually anymore, just reference `@item.Partition` etc.